apps/frontend/src/pages/analytics/analytics.tsx

```diff
@@ -22,26 +24,34 @@ const Analytics = (): JSX.Element => {
 	const { activityLogs, dataStatus } = useAppSelector(
 		({ activityLogs }) => activityLogs,
 	);
+	const { projects } = useAppSelector(({ projects }) => projects);
+
+	useEffect(() => {
+		void dispatch(projectActions.loadAll());
```

Comment:
We need to load these projects separately, specifically for analytics module, so it should be in analytics slice. Also on the backend side we need to split service method for pagination and without and the same for repository. Take a look at how it's done here #491

Reply:
Did I understand correctly that I can make a separate request on the backend in order to get all the projects (without pagination)? And use it in the analytics slice? I thought it would be better that way, I asked, but I didn't get an answer then

Reply:
Yes, let's make it as a separate request

Reply:
probably missed your question in the chat

Reply:
Everything is fine, I hope it is correct now